### ISSUE TITLE
Language Client: Show language server status in status bar

### ISSUE THREAD

**Issue by extr0py** [extr0py/oni#383]
Dependent on #201 - but it's helpful to get an update of what the language server is doing. In some cases, like Omnisharp, the initial load can be slow since it downloads a binary - it'd be helpful to have an indicator / spinner / error icon to show the state of the language server.

**Comment by Galbar** [comment]
Is this a duplicate of #485 ?

**Comment by extr0py** [comment]
Derp... this sure is a duplicate! Thanks for catching it, @Galbar ! I'll close this one out since #485 has a bit more info.
